src/goose/brokers/redis/batch.clj

```diff
@@ -0,0 +1,30 @@
+(ns goose.brokers.redis.batch
+  (:require
+    [goose.brokers.redis.commands :as redis-cmd]
+    [goose.defaults :as d]
+    [goose.job :as job]
+    [goose.retry :as retry]))
+
+(defn update-state [next]
+  (fn [{:keys [redis-conn] :as opts}
+       {:keys [id, batch-id] :as job}]
+    (if batch-id
+      (let [response (next opts job)
```

Comment:
suggestion: this needs to be in the try catch block. This is the main place where we are trying to see errors etc